### PR TITLE
49934 : Fix Problem when opening the qr camera for first time

### PR DESCRIPTION
**Pull request by wbenabdessalem**
When i opens Android first camera for QR code at first time, and error message is displayed (Barcode Scanner)
When i open it the second time, it works without problems.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -7,7 +7,6 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-feature android:name="android.hardware.camera.any" />
     <uses-permission android:name="android.permission.CAMERA" />
-
```
Comment (ahamdi) [inline app/src/main/AndroidManifest.xml]: Could you  revert this change ?

Comment (wbenabdessalem) [reply app/src/main/AndroidManifest.xml]: Ok

**Review by ahamdi — APPROVED** [review]
